packaging/ubuntu-14.04/rules

```diff
@@ -128,6 +128,13 @@ override_dh_auto_build:
 	mkdir -p _build/src/$(DH_GOPKG)/cmd/snap/test-data
 	cp -a cmd/snap/test-data/*.gpg _build/src/$(DH_GOPKG)/cmd/snap/test-data/
 	dh_auto_build -- $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS)
+	# Generate static snap-exec - it somehow includes CGO so we must
+	# force a static build here. We need a static snap-exec inside
+	# the core snap because not all bases will have a libc
```

Comment:
:heart:

Reply:
The base field is on the app snap itself, while the base type is on the underlying snap. These sound special enough to be worth having a new type for them. We want to hide them in the default listing, for example.